types.ts

```diff
@@ -28,11 +28,11 @@ export interface FeatureData {
     compat_features?: string[];
 }
 
-type browserIdentifier = "chrome" | "chrome_android" | "edge" | "firefox" | "firefox_android" | "safari" | "safari_ios";
+export type BrowserIdentifier = "chrome" | "chrome_android" | "edge" | "firefox" | "firefox_android" | "safari" | "safari_ios";
 
-type BaselineHighLow = "high" | "low";
+export type BaselineHighLow = "high" | "low";
 
-interface Status {
+export interface Status {
```

Comment:
I think this is a little weird. `Status` + `SupportStatus` is a little bit of a hack to avoid recursion in the generated JSON Schema (and I don't want to suggest that the `by_compat_key` data is a tree) or writing mostly duplicate types. I'd suggest exporting only `SupportStatus` and see what this looks like after regenerating the schema.